fah_alchemy/compute/api.py

```diff
@@ -63,6 +63,22 @@ async def info():
     return {"message": "nothing yet"}
 
 
+@router.get("/check")
+async def check(
+    n4js: Neo4jStore = Depends(get_n4js_depends),
+    s3os: S3ObjectStore = Depends(get_s3os_depends),
+):
+    # check if neo4j database is reachable
+    neo4jreachable = n4js._store_check()
+    # check if s3 object store is reachable
+    s3reachable = s3os._store_check()
+    if not neo4jreachable or not s3reachable:
+        code = status.HTTP_503_SERVICE_UNAVAILABLE
+    else:
+        code = status.HTTP_200_OK
+    return {"neo4jreachable": neo4jreachable, "s3reachable": s3reachable, "code": code}
```

Comment:
@dotsdl This is returning a status code as part of a JSON payload, I know this is probably not the right way to do it but wasn't exactly sure what the best way was to return a payload and status code so that `requests` picks it up.

Reply:
To elaborate I mean so that it is then accessible with `response.status_code.`